Task4

ReshetovItsMe opened the pull request:
All tasks are fully done
[https://wguce6f2l7.execute-api.eu-west-1.amazonaws.com/dev/products](https://wguce6f2l7.execute-api.eu-west-1.amazonaws.com/dev/products)
[https://wguce6f2l7.execute-api.eu-west-1.amazonaws.com/dev/products/7567ec4b-b10c-48c5-9345-fc73c48a80aa](https://wguce6f2l7.execute-api.eu-west-1.amazonaws.com/dev/products/7567ec4b-b10c-48c5-9345-fc73c48a80aa)

and post method [https://wguce6f2l7.execute-api.eu-west-1.amazonaws.com/dev/products/](https://wguce6f2l7.execute-api.eu-west-1.amazonaws.com/dev/products/)

body example for postman: 
`{
    "id": "{{$guid}}",
    "title": "test 121",
    "description": "TEST description 1210",
    "price": 13110,
    "count": 12110
}`

cloudfront [d1nn7ah5azefhq.cloudfront.net](https://d1nn7ah5azefhq.cloudfront.net/)
[FRONT END PR ](https://github.com/ReshetovItsMe/shop-react-redux-cloudfront/pull/3)

alimovdavron commented:
![image](https://user-images.githubusercontent.com/37940444/127969718-5ea8f465-74ce-4ff8-8dac-5faed0d2fe91.png)
5 - Your own Frontend application is integrated with product service (/products API) and products from product-service are represented on Frontend. Link to a working Front-End application is provided for cross-check reviewer. - the task is not implemented. I'm not able to add product using frontend form

ReshetovItsMe commented:
> ![image](https://user-images.githubusercontent.com/37940444/127969718-5ea8f465-74ce-4ff8-8dac-5faed0d2fe91.png)
> 5 - Your own Frontend application is integrated with product service (/products API) and products from product-service are represented on Frontend. Link to a working Front-End application is provided for cross-check reviewer. - the task is not implemented. I'm not able to add product using frontend form

Fixed and updated link to FE PR in description
